packages/components/src/progress-bar/stories/index.story.tsx

```diff
@@ -24,9 +24,36 @@ const meta: Meta< typeof ProgressBar > = {
 };
 export default meta;
 
-const Template: StoryFn< typeof ProgressBar > = ( { ...args } ) => {
-	return <ProgressBar { ...args } />;
-};
+const Template: StoryFn< typeof ProgressBar > = ( { ...args } ) => (
+	<ProgressBar { ...args } />
+);
 
 export const Default: StoryFn< typeof ProgressBar > = Template.bind( {} );
 Default.args = {};
+
+export const ParentWidth: StoryFn< { parentWidth: string } > = ( {
```

Comment:
> I'm not sure that's the right place for a "parent width" control (considering it's not a prop of the actual component), but I think it works nicely to show/document it and allow the user to see it in practice.

Your concerns are spot on — in our Storybook, we've decided to have our controls maintain a strict 1:1 relationship with the actual props, for clarity and consistency. So we don't want a control that's not an actual prop.

But I do think there should be a story like this to show the suggested default width (160px).

Reply:
OK, if the goal is to just show a default width, I'll simplify the story to just use a custom `className` that sets it to 160px. We can then improve the docs to show the different ways of setting the width.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/61062/commits/5e03b5254ea2dbfcfce6e37a943fde492c2af7c5. It now looks like this:

<img width="1493" alt="Screenshot 2024-05-23 at 12 35 39 p m" src="https://github.com/WordPress/gutenberg/assets/81248/063f00e5-9874-49f7-a094-6a3e15f1cb9a">